### ISSUE TITLE
"message":"Error updating Test: test with id 30281 not found!

### ISSUE THREAD

**Issue by dcardozoo** [Xray-App/xray-cloud-mig#2]
Hi Team, 

I'm currently migration XRay from server to Cloud but when Syncing the Manual test definitions I got an error while commiting into the Jira cloud. 

```python3 xray_syncManualTestsDefinitions.py -rerun
2021-01-14 22:54:08,168 - root - INFO - ------------------- ##### Starting ##### -------------------
...
2021-01-14 22:54:11,583 - root - INFO - -------
2021-01-14 22:54:11,583 - root - INFO - ======= Collecting INFO [START] =========
2021-01-14 22:54:11,583 - root - INFO - Processing Tests...
2021-01-14 22:54:18,617 - root - INFO - Cleaning FT-478
2021-01-14 22:54:18,964 - root - INFO - ======= Collecting INFO [STOP] =========
2021-01-14 22:54:18,964 - root - INFO - ======= Persisting information in Cloud [START] =========
2021-01-14 22:54:18,964 - root - INFO - Pack mutation and push...
2021-01-14 22:54:18,964 - root - INFO - Commiting...
2021-01-14 22:54:20,273 - root - INFO - Ohh dear...
2021-01-14 22:54:20,274 - root - INFO - {"errors":[{"message":"Error updating Test: test with id 30281 not found!","locations":[{"line":1,"column":13}],"path":["ztjmtyohit"]}],"data":{"ztjmtyohit":null}}
2021-01-14 22:54:20,274 - root - INFO - Ohh dear...
2021-01-14 22:54:20,274 - root - INFO - {"errors":[{"message":"Error updating Test: test with id 30281 not found!","locations":[{"line":1,"column":13}],"path":["ztjmtyohit"]}],"data":{"ztjmtyohit":null}}```

**XRay Cloud version: 1.1.51-AC**
**XRay OnPremise version: *4.2.5*

**Comment by lfcribeiro** [comment]
Hi,

Have your follow all the steps to make sure that the migrated tests are being recognized as Xray Tests in the Cloud ?
Have you performed the Xray reindex of the project https://docs.getxray.app/display/XRAYCLOUD/Project+Settings%3A+Re-Indexing ?

Kind regards,

**Comment by dcardozoo** [comment]
Hi @lfcribeiro, it was an issue whith IssueTypes mapping. 
After fixing this and re-index, it worked just fine!. Nevertheless the documentation is very confusing and hard to follow. 

Best regards,
